src/fidesops/ops/service/connectors/saas/connector_registry_service.py

```diff
@@ -133,3 +140,129 @@ def load_registry(config_file: str) -> ConnectorRegistry:
     if _registry is None:
         _registry = ConnectorRegistry.parse_obj(load_toml([config_file]))
     return _registry
+
+
+def update_saas_configs(registry: ConnectorRegistry, db: Session) -> None:
+    """
+    Updates SaaS config instances currently in the DB if to the
+    corresponding template in the registry are found.
+
+    Effectively an "update script" for SaaS config instances,
+    to be run on server bootstrap.
+    """
+    for connector_type in registry.connector_types():
+        logger.debug(
+            "Determining if any updates are needed for connectors of type %s based on templates...",
+            connector_type,
+        )
+        template: ConnectorTemplate = registry.get_connector_template(  # type: ignore
+            connector_type
+        )
+        saas_config_template = SaaSConfig.parse_obj(load_config(template.config))
+        template_version: Union[LegacyVersion, Version] = parse_version(
```

Comment:
Unfamiliar with these LegacyVersion versus Versions here, what is this?

Reply:
this is just the return type coming from `packaging.version.parse` - i needed to include the union type here to satisfy mypy (or at least that was what i thought was the simplest way to satisfy)